source/frontend/carla_host.py

```diff
@@ -1618,18 +1618,34 @@ def slot_handlePatchbayPortRemovedCallback(self, groupId, portId):
 
     @pyqtSlot(int, int, int, int, str)
     def slot_handlePatchbayPortChangedCallback(self, groupId, portId, portFlags, portGroupId, newPortName):
-        patchcanvas.renamePort(groupId, portId, newPortName)
+        patchcanvas.changePortProperties(groupId, portId, portGroupId, newPortName)
         self.updateMiniCanvasLater()
 
     @pyqtSlot(int, int, int, str)
-    def slot_handlePatchbayPortGroupAddedCallback(self, groupId, portId, portGroupId, newPortName):
-        # TODO
-        pass
+    def slot_handlePatchbayPortGroupAddedCallback(self, groupId, portFlags, portGroupId, newPortName):
```

Comment:
I dont understand why the change of argument order was needed here..

Reply:
oh wait, I understand, this is adding flags instead of port-id which seems useless for you.
please rename portFlags to portGroupFlags then, thanks